models/prices/base/prices_base_tokens.sql

```diff
@@ -0,0 +1,24 @@
+{{ config(
+        schema='prices_base',
+        alias = alias('tokens'),
+        materialized='table',
+        file_format = 'delta',
+        tags=['static', 'dunesql']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+
+--    ('weth-weth', 'base', 'WETH', 0x4200000000000000000000000000000000000006, 18), --requested add to coinPaprika 2023-08-10
+--    ('axl-axelar', 'base', 'AXL', 0x467719aD09025FcC6cF6F8311755809d45a5E5f3, 6), --requested add to coinPaprika 2023-08-10
+    ('bald-bald', 'base', 'BALD', 0x27D2DECb4bFC9C76F0309b8E88dec3a601Fe25a8, 18)
```

Comment:
SCAN: https://base.blockscout.com/address/0x27D2DECb4bFC9C76F0309b8E88dec3a601Fe25a8
API: https://api.coinpaprika.com/v1/coins/bald-bald

Reply:
However currently base chain is not supported in coinpaprika yet:
https://api.coinpaprika.com/v1/contracts/

We may need to wait for a while.

Reply:
i think it's fine as long as the token_id returns a response and we manually assign address here

Reply:
note that the token is invalid on the API though, so we won't get ongoing pricing data